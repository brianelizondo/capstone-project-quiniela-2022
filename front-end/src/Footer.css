.Footer{
    color: white;
    text-align: center;
    font-size: 12px;
    background-color: rgba(0, 0, 0, .65);
    border-top: 5px solid rgba(0, 0, 0, .80);
    padding: 20px 0px;
    margin-top: 50px;
}
.Footer a{
    text-decoration: none;
    color: white;
    font-weight: 700;
}
.Footer a:hover{
    color: rgb(60, 154, 236);
}