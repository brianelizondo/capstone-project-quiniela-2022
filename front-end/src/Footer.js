import React from 'react';
import './Footer.css';

function Footer(){
    return (
        <div className="Footer">
            Quinielas World Cup 2022. Developed by <a href="https://github.com/brianelizondo">Brian Elizondo</a> for the USF Bootcamp Capstone Project
        </div>
    );
}

export default Footer;