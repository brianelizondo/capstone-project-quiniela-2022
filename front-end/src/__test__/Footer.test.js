import React from 'react';
import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom';

import Footer from '../Footer';

// Smoke Test
describe("Smoke Test", () => {
    it("renders 'Footer' component without crashing", () => {
        render(<Footer />);
        const textFound = screen.getByText("Brian Elizondo");
        expect(textFound).toBeInTheDocument();
    });
});

// Snapshot Tests
describe("Snapshot Test", () => {
    it("'Footer' matches snapshot", () => {
        const {asFragment} = render(<Footer />);
        expect(asFragment()).toMatchSnapshot();
    });
});