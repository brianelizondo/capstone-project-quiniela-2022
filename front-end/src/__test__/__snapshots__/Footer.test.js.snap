// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot Test 'Footer' matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="Footer"
  >
    Quinielas World Cup 2022. Developed by 
    <a
      href="https://github.com/brianelizondo"
    >
      Brian Elizondo
    </a>
     for Capstone Project
  </div>
</DocumentFragment>
`;
